diff --git a/conf/machine/starfive-dubhe.conf b/conf/machine/starfive-dubhe.conf
index a6c7b9d..bba4e61 100644
--- a/conf/machine/starfive-dubhe.conf
+++ b/conf/machine/starfive-dubhe.conf
@@ -5,7 +5,7 @@
 
 LLVMVERSION = "14.0.0"
 CLANGSDK = "1"
-EXTRA_IMAGEDEPENDS:append = " llvm"
+# EXTRA_IMAGEDEPENDS:append = " llvm"
 EXTRA_IMAGE_FEATURES:append = " dbg-pkgs tools-profile"
 
 PACKAGE_DEBUG_SPLIT_STYLE = 'debug-file-directory'
